### DISCUSSION TITLE
Raspberry 5 + Sailor Hat Version 2.0.0 + Openplotter 4 / Bookworm: Missing /etc/shrpid.conf

### DISCUSSION THREAD

**Discussion by maix0000** [hatlabs/discussions#119] [General]
Raspberry 5 + Sailor Hat Version 2.0.0 + Openplotter 4 / Bookworm: Missing /etc/shrpid.conf

Installation went without errors.
However, there is no config file "/etc/shrpid.conf".
What am I missing?

**Comment by HPN-1956** [comment]
I can only add to this issue that it is the same problem on a RPI 4.

**Comment by mairas** [comment]
Hi,

Sorry about not catching this sooner.

The wording in the README is a bit misleading - there indeed is no default config file, but you need to create one yourself. I added a section about the config file in the documentation: https://docs.hatlabs.fi/sh-rpi/docs/software/#daemon-configuration-file
